doc/man3/SSL_set_connect_state.pod

```diff
@@ -39,11 +44,19 @@ using L<SSL_write_ex(3)>, L<SSL_write(3)>, L<SSL_read_ex(3)>, or L<SSL_read(3)>,
 the handshake routines must be explicitly set in advance using either
 SSL_set_connect_state() or SSL_set_accept_state().
 
+If SSL_is_server() is called before SSL_set_connect_state() or
+SSL_set_accept_state() is called (either automatically or explicitly),
+the result depends on what method is used when SSL_CTX is created with
+L<SSL_CTX_new(3)>. If a generic method or a dedicated server method is
+passed to L<SSL_CTX_new(3)>, SSL_is_server() returns 1 otherwise it returns 0.
```

Comment:
I am really not familiar with the usage of ';' in English, is it also acceptable as "returns1, otherwise it returns 0"...